modules/images-configuration-parameters.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration.adoc
+
+[id="images-configuration-parameters_{context}"]
+= Image controller configuration parameters
+
+The `image.config.openshift.io/cluster` resource offers the following
+configuration parameters.
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`metadata`
+|Standard object's metadata.
+
+|`spec`
+|Holds user-settable values for configuration.
+
+|`status`
+|Holds observed values from the cluster.
+
+|`allowedRegistriesForImport`
+|Limits the container image registries from which normal users may import images.
+Set this list to the registries that you trust to contain valid Docker images,
+and that you want applications to be able to import from. Users with permission
+to create images or `ImageStreamMappings` from the API are not affected by this
+policy. Typically only cluster administrators will have the appropriate permissions.
+
+|`additionalTrustedCA`
+|A reference to a ConfigMap containing additional CAs that should be trusted
+during ImageStream import, pod image pull, and `openshift-image-registry`
+pullthrough.
+
+The namespace for this ConfigMap is `openshift-config`.
```

Comment:
"The format of the configmap is to use the registry hostname as the key, and the base64-encoded certificate as the value, for each additional registry CA to trust"